qa/vagrant-helpers.rb

```diff
@@ -34,10 +50,17 @@ def self.parse(lines)
 
     def self.execute(cmd)
       Open3.popen3(cmd) do |stdin, stdout, stderr, wait_thr|
-        { :stdout => stdout.read.chomp, :stderr => stderr.read.chomp,
-          :exit_status => wait_thr.value.exitstatus }
+        ExecuteResponse.new(stdin, stdout.read.chomp, stderr.read.chomp, wait_thr.value.exitstatus)
```

Comment:
Well wrapping this in a class can allow us to iterate on the response object, It could have been a struct. It just easier to refactor than using a hash.